rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -258,6 +258,7 @@ def test_add_delete_xpub(rotkehlchen_api_server):
     assert len(result) == 0, 'all xpub mappings should have been deleted'
 
 
+@pytest.mark.vcr
```

Comment:
This involves balance queries on-chain. This is not repeatable.

Reply:
Okay as discussed in the call let's go with this. Use vcr anywhere we can and avoid any complicated mocks.